woo-razorpay.php

```diff
@@ -818,11 +818,16 @@ protected function verifyOrderAmount($razorpayOrderId, $orderId)
 
             $razorpayOrderArgs = array(
                 'id'        => $razorpayOrderId,
-                'amount'    => $orderCreationData['amount'],
                 'currency'  => $orderCreationData['currency'],
                 'receipt'   => (string) $orderId,
             );
 
+            if(is1ccEnabled()){
```

Comment:
@poojarazor don't use this flag for condition, instead of that use `is1ccCheckout` by passing in params.